### ISSUE TITLE
Preset loading not working when starting as a service

### ISSUE THREAD

**Issue by Janrupf** [wwmm/easyeffects#3010]
### EasyEffects Version

7.1.5

### What package are you using?

NixOS

### Distribution

NixOS

### Describe the bug

Running `easyeffects --gapplication-service -l my-preset` doesn't load the preset `my-preset`. Separately running `easyeffects -l my-preset` once the daemon is running works correctly. 

### Expected Behavior

Load presets specified with `-l`/`--load-preset` even when starting with `--gapplication-service`.

### Debug Log

<details><summary>Debug Log</summary><br/>

```
(process:5241): easyeffects-DEBUG: 19:41:59.254: 	easyeffects.cpp:46	easyeffects version: 7.1.5
(process:5241): easyeffects-DEBUG: 19:41:59.255: 	easyeffects.cpp:56	locale directory: /nix/store/3ln2acvn5056irhw44lzk4dzghbrw373-easyeffects-7.1.5/share/locale
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.257: 	presets_manager.cpp:212	user presets directory already exists: /home/janrupf/.config/easyeffects/
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.257: 	presets_manager.cpp:212	user presets directory already exists: /home/janrupf/.config/easyeffects/input
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.257: 	presets_manager.cpp:212	user presets directory already exists: /home/janrupf/.config/easyeffects/output
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.257: 	presets_manager.cpp:212	user presets directory already exists: /home/janrupf/.config/easyeffects/autoload/input
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.257: 	presets_manager.cpp:212	user presets directory already exists: /home/janrupf/.config/easyeffects/autoload/output
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.436: 	pipe_manager.cpp:1534	compiled with PipeWire: 1.0.4
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.436: 	pipe_manager.cpp:1535	linked to PipeWire: 1.0.4
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.443: 	pipe_manager.cpp:1505	core version: 1.0.4
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.443: 	pipe_manager.cpp:1506	core name: pipewire-0
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.443: 	pipe_manager.cpp:1417	found metadata: settings
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: schema-sm-settings
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: persistent-sm-settings
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: sm-settings
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: sm-objects
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: default
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: filters
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1417	found metadata: route-settings
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Audio/Sink 87 alsa_output.pci-0000_09_00.1.hdmi-stereo with serial 87 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Audio/Sink 91 alsa_output.pci-0000_0b_00.3.iec958-stereo with serial 91 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Audio/Source 92 alsa_input.pci-0000_0b_00.3.analog-stereo with serial 92 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Audio/Sink 89 alsa_output.usb-0ccd_0077-00.analog-stereo with serial 101 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Audio/Source 88 alsa_input.usb-0ccd_0077-00.mono-fallback with serial 102 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Audio/Source 90 alsa_input.usb-046d_BRIO_4K_Stream_Edition_D4E1A92C-02.analog-stereo with serial 103 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 78 PulseAudio Volume Control with serial 139 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 77 PulseAudio Volume Control with serial 140 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 76 PulseAudio Volume Control with serial 141 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 75 PulseAudio Volume Control with serial 142 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 123 PulseAudio Volume Control with serial 143 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 122 PulseAudio Volume Control with serial 144 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_output.pci-0000_09_00.1.hdmi-stereo port 102 is connected to PulseAudio Volume Control port 138
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_output.pci-0000_09_00.1.hdmi-stereo port 104 is connected to PulseAudio Volume Control port 140
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_output.pci-0000_0b_00.3.iec958-stereo port 106 is connected to PulseAudio Volume Control port 144
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_output.pci-0000_0b_00.3.iec958-stereo port 108 is connected to PulseAudio Volume Control port 146
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_input.pci-0000_0b_00.3.analog-stereo port 109 is connected to PulseAudio Volume Control port 150
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_input.pci-0000_0b_00.3.analog-stereo port 110 is connected to PulseAudio Volume Control port 152
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_output.usb-0ccd_0077-00.analog-stereo port 112 is connected to PulseAudio Volume Control port 156
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_output.usb-0ccd_0077-00.analog-stereo port 114 is connected to PulseAudio Volume Control port 158
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_input.usb-0ccd_0077-00.mono-fallback port 115 is connected to PulseAudio Volume Control port 162
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_input.usb-046d_BRIO_4K_Stream_Edition_D4E1A92C-02.analog-stereo port 116 is connected to PulseAudio Volume Control port 165
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_input.usb-046d_BRIO_4K_Stream_Edition_D4E1A92C-02.analog-stereo port 117 is connected to PulseAudio Volume Control port 167
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Output/Audio 225 Chromium with serial 249 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 236 PulseAudio Volume Control with serial 267 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 243 Chromium input with serial 274 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 53 PulseAudio Volume Control with serial 1213 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 137 PulseAudio Volume Control with serial 1215 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1311	alsa_input.usb-0ccd_0077-00.mono-fallback port 115 is connected to PulseAudio Volume Control port 228
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.444: 	pipe_manager.cpp:1271	Stream/Input/Audio 223 PulseAudio Volume Control with serial 1279 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1271	Stream/Input/Audio 45 PulseAudio Volume Control with serial 1287 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	alsa_input.usb-0ccd_0077-00.mono-fallback port 115 is connected to PulseAudio Volume Control port 222
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	alsa_input.usb-0ccd_0077-00.mono-fallback port 115 is connected to PulseAudio Volume Control port 234
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	alsa_input.usb-0ccd_0077-00.mono-fallback port 115 is connected to Chromium input port 178
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	Chromium port 226 is connected to alsa_output.usb-0ccd_0077-00.analog-stereo port 111
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	Chromium port 227 is connected to alsa_output.usb-0ccd_0077-00.analog-stereo port 113
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1271	Stream/Input/Audio 230 PulseAudio Volume Control with serial 1421 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	Chromium port 226 is connected to PulseAudio Volume Control port 197
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1311	Chromium port 227 is connected to PulseAudio Volume Control port 185
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1271	Audio/Sink 121 easyeffects_sink with serial 1429 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1271	Audio/Source/Virtual 44 easyeffects_source with serial 1430 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 0, default.configured.audio.source, Spa:String:JSON, { "name": "alsa_input.usb-0ccd_0077-00.mono-fallback" }
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 0, default.audio.sink, Spa:String:JSON, {"name":"alsa_output.usb-0ccd_0077-00.analog-stereo"}
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 0, default.audio.source, Spa:String:JSON, {"name":"alsa_input.usb-0ccd_0077-00.mono-fallback"}
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 0, default.video.source, Spa:String:JSON, {"name":"v4l2_input.pci-0000_0a_00.3-usb-0_2.1.3_1.0"}
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 225, target.node, Spa:Id, 52
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 225, target.object, Spa:Id, 1295
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 243, target.node, Spa:Id, 220
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.445: 	pipe_manager.cpp:1007	new metadata property: 243, target.object, Spa:Id, 1296
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:1633	easyeffects_sink node successfully retrieved with id 121 and serial 1429
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:1638	easyeffects_source node successfully retrieved with id 44 and serial 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:394	Stream/Input/Audio 78 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:394	Stream/Input/Audio 77 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:394	Stream/Input/Audio 76 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:394	Stream/Input/Audio 75 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:394	Stream/Input/Audio 123 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.446: 	pipe_manager.cpp:394	Stream/Input/Audio 122 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.447: 	pipe_manager.cpp:394	Stream/Input/Audio 236 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.447: 	pipe_manager.cpp:394	Stream/Input/Audio 53 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.447: 	pipe_manager.cpp:394	Stream/Input/Audio 137 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.447: 	pipe_manager.cpp:394	Stream/Input/Audio 223 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.447: 	pipe_manager.cpp:394	Stream/Input/Audio 45 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.447: 	pipe_manager.cpp:394	Stream/Input/Audio 230 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.452: 	plugin_base.cpp:403	soe: output_level successfully connected to PipeWire graph
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.454: 	plugin_base.cpp:403	soe: spectrum successfully connected to PipeWire graph
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.454: 	pipe_manager.cpp:1314	No node with id 135 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.455: 	pipe_manager.cpp:1314	No node with id 135 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.456: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.456: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.456: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.457: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.460: 	plugin_base.cpp:403	sie: output_level successfully connected to PipeWire graph
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.462: 	plugin_base.cpp:403	sie: spectrum successfully connected to PipeWire graph
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.462: 	pipe_manager.cpp:1314	No node with id 128 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.463: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.463: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.464: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.464: 	output_level.cpp:44	soe: output_level: PipeWire blocksize: 512
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.464: 	output_level.cpp:45	soe: output_level: PipeWire sampling rate: 48000
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.464: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.466: 	pipe_manager.cpp:1271	Stream/Input/Audio 130 PulseAudio Volume Control with serial 1470 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.466: 	pipe_manager.cpp:330	The input stream PulseAudio Volume Control does not have as target the same mic used as EE input: alsa_input.usb-0ccd_0077-00.mono-fallback
 The user wants it to record from device 1429. We will ignore this stream.
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.466: 	pipe_manager.cpp:394	Stream/Input/Audio 130 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.468: 	pipe_manager.cpp:1314	No node with id 135 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.468: 	pipe_manager.cpp:1314	No node with id 135 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.469: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.469: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.470: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.470: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.473: 	pipe_manager.cpp:1314	No node with id 128 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.473: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.474: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.474: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.475: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
easyeffects-INFO: 19:41:59.475: 	application.cpp:76	disabling global bypass
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.475: 	pipe_manager.cpp:1271	Stream/Input/Audio 36 PulseAudio Volume Control with serial 1482 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.475: 	pipe_manager.cpp:330	The input stream PulseAudio Volume Control does not have as target the same mic used as EE input: alsa_input.usb-0ccd_0077-00.mono-fallback
 The user wants it to record from device 1430. We will ignore this stream.
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.476: 	pipe_manager.cpp:394	Stream/Input/Audio 36 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.482: 	pipe_manager.cpp:1314	No node with id 135 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.482: 	pipe_manager.cpp:1314	No node with id 135 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.483: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.483: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.484: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.484: 	pipe_manager.cpp:1314	No node with id 65 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.487: 	pipe_manager.cpp:1314	No node with id 128 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.488: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.488: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.489: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.489: 	pipe_manager.cpp:1314	No node with id 79 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.491: 	application.cpp:104	new default output device: alsa_output.usb-0ccd_0077-00.analog-stereo
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.491: 	application.cpp:112	new default input device: alsa_input.usb-0ccd_0077-00.mono-fallback
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:156	device alsa_card.pci-0000_09_00.1 has changed its output route to: hdmi-output-0
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:176	output autoloading: the target node name does not match the output device name
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:124	device alsa_card.usb-0ccd_0077-00 has changed its input route to: analog-input-mic
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:156	device alsa_card.usb-0ccd_0077-00 has changed its output route to: analog-output-speaker
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:124	device alsa_card.usb-046d_BRIO_4K_Stream_Edition_D4E1A92C-02 has changed its input route to: analog-input-mic
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:144	input autoloading: the target node name does not match the input device name
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:156	device alsa_card.pci-0000_0b_00.3 has changed its output route to: iec958-stereo-output
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.492: 	application.cpp:176	output autoloading: the target node name does not match the output device name
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 78, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 78, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 77, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 77, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 76, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 76, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 75, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 75, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 123, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 123, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 122, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 122, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 225, target.node, Spa:Id, 121
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 225, target.object, Spa:Id, 1429
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 236, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 236, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 243, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 243, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 53, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 53, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 137, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 137, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 223, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 223, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 45, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 45, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 230, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 230, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 78, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 78, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 77, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 77, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 76, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 76, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 75, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 75, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 123, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 123, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 122, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 122, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 236, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 236, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 53, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 53, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 137, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 137, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 223, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 223, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 45, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 45, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 230, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.515: 	pipe_manager.cpp:1007	new metadata property: 230, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 130, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 130, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 130, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 130, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 36, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 36, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 36, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.516: 	pipe_manager.cpp:1007	new metadata property: 36, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.517: 	pipe_manager.cpp:1314	No node with id 130 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.517: 	pipe_manager.cpp:1314	No node with id 130 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.522: 	pipe_manager.cpp:1314	No node with id 36 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.522: 	pipe_manager.cpp:1314	No node with id 36 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.527: 	output_level.cpp:44	sie: output_level: PipeWire blocksize: 256
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.527: 	output_level.cpp:45	sie: output_level: PipeWire sampling rate: 48000
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.556: 	pipe_manager.cpp:1311	easyeffects_source port 73 is connected to Chromium input port 124
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.556: 	pipe_manager.cpp:1311	easyeffects_source port 220 is connected to Chromium input port 136
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.567: 	pipe_manager.cpp:1311	Chromium port 226 is connected to easyeffects_sink port 211
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.567: 	pipe_manager.cpp:1311	Chromium port 227 is connected to easyeffects_sink port 213
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.572: 	pipe_manager.cpp:1271	Stream/Input/Audio 230 PulseAudio Volume Control with serial 1514 has been added
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.572: 	pipe_manager.cpp:330	The input stream PulseAudio Volume Control does not have as target the same mic used as EE input: alsa_input.usb-0ccd_0077-00.mono-fallback
 The user wants it to record from device 249. We will ignore this stream.
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.572: 	pipe_manager.cpp:394	Stream/Input/Audio 230 PulseAudio Volume Control has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.579: 	pipe_manager.cpp:1007	new metadata property: 230, target.node, Spa:Id, 44
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.579: 	pipe_manager.cpp:1007	new metadata property: 230, target.object, Spa:Id, 1430
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.579: 	pipe_manager.cpp:1007	new metadata property: 230, target.node, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.579: 	pipe_manager.cpp:1007	new metadata property: 230, target.object, , 
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.626: 	pipe_manager.cpp:1314	No node with id 230 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:41:59.627: 	pipe_manager.cpp:1314	No node with id 230 in our node_map
(easyeffects:5241): easyeffects-DEBUG: 19:42:01.491: 	application_ui.cpp:109	Icon Theme breeze-dark detected
(easyeffects:5241): easyeffects-DEBUG: 19:42:01.724: 	app_info.cpp:178	cannot lookup application icon chromium in /usr/share/pixmaps
(easyeffects:5241): easyeffects-DEBUG: 19:42:01.724: 	app_info.cpp:178	cannot lookup application icon chromium in /usr/local/share/pixmaps
(easyeffects:5241): easyeffects-DEBUG: 19:42:01.728: 	app_info.cpp:178	cannot lookup application icon chromium in /usr/share/pixmaps
(easyeffects:5241): easyeffects-DEBUG: 19:42:01.728: 	app_info.cpp:178	cannot lookup application icon chromium in /usr/local/share/pixmaps

(easyeffects:5241): Gtk-WARNING **: 19:42:01.794: GtkGizmo 0x294b710 (slider) reported min width -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:01.794: GtkGizmo 0x2949df0 (slider) reported min height -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:04.406: GtkGizmo 0x29a5960 (slider) reported min width -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:04.406: GtkGizmo 0x29a3f80 (slider) reported min height -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:05.273: GtkGizmo 0x29b7440 (slider) reported min width -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:05.274: GtkGizmo 0x29b5a60 (slider) reported min height -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:05.274: GtkGizmo 0x29c0020 (slider) reported min width -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:05.274: GtkGizmo 0x29be540 (slider) reported min height -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:05.280: GtkGizmo 0x29c9a10 (slider) reported min width -2, but sizes must be >= 0

(easyeffects:5241): Gtk-WARNING **: 19:42:05.280: GtkGizmo 0x29c7ed0 (slider) reported min height -2, but sizes must be >= 0
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.666: 	presets_menu.cpp:494	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.666: 	presets_menu.cpp:44	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.666: 	presets_menu.cpp:504	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.666: 	application_ui.cpp:262	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.667: 	app_info.cpp:377	Chromium disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.667: 	app_info.cpp:385	Chromium finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.667: 	app_info.cpp:63	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.668: 	app_info.cpp:377	Chromium input disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.668: 	app_info.cpp:385	Chromium input finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.668: 	app_info.cpp:63	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.672: 	presets_menu.cpp:494	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.672: 	presets_menu.cpp:44	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.672: 	presets_menu.cpp:504	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.672: 	effects_box.cpp:516	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.672: 	chart.cpp:49	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.672: 	chart.cpp:652	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	blocklist_menu.cpp:219	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	blocklist_menu.cpp:52	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	blocklist_menu.cpp:229	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	apps_box.cpp:414	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	node_info_holder.cpp:106	225, Chromium finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	apps_box.cpp:51	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	apps_box.cpp:424	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.673: 	plugins_box.cpp:849	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	plugins_menu.cpp:272	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	plugins_menu.cpp:52	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	plugins_menu.cpp:282	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	plugins_box.cpp:112	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	plugins_box.cpp:859	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	effects_box.cpp:61	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	effects_box.cpp:526	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	effects_box.cpp:516	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.674: 	chart.cpp:49	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.675: 	chart.cpp:652	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.675: 	blocklist_menu.cpp:219	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.675: 	blocklist_menu.cpp:52	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.675: 	blocklist_menu.cpp:229	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.675: 	apps_box.cpp:414	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.675: 	node_info_holder.cpp:106	243, Chromium input finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.676: 	apps_box.cpp:51	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.676: 	apps_box.cpp:424	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.676: 	plugins_box.cpp:849	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.677: 	plugins_menu.cpp:272	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	plugins_menu.cpp:52	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	plugins_menu.cpp:282	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	plugins_box.cpp:112	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	plugins_box.cpp:859	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	effects_box.cpp:61	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	effects_box.cpp:526	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.678: 	pipe_manager_box.cpp:839	disposed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.682: 	node_info_holder.cpp:106	87, alsa_output.pci-0000_09_00.1.hdmi-stereo finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.682: 	node_info_holder.cpp:106	91, alsa_output.pci-0000_0b_00.3.iec958-stereo finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.682: 	node_info_holder.cpp:106	89, alsa_output.usb-0ccd_0077-00.analog-stereo finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.682: 	node_info_holder.cpp:106	92, alsa_input.pci-0000_0b_00.3.analog-stereo finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.682: 	node_info_holder.cpp:106	90, alsa_input.usb-046d_BRIO_4K_Stream_Edition_D4E1A92C-02.analog-stereo finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.682: 	node_info_holder.cpp:106	88, alsa_input.usb-0ccd_0077-00.mono-fallback finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.683: 	pipe_manager_box.cpp:57	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.683: 	test_signals.cpp:215	destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.683: 	pipe_manager_box.cpp:849	finalized
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.683: 	application_ui.cpp:52	data struct destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:07.683: 	application_ui.cpp:272	finalized
^C(easyeffects:5241): easyeffects-DEBUG: 19:42:09.412: 	presets_manager.cpp:207	destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.415: 	stream_input_effects.cpp:134	destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.415: 	effects_base.cpp:152	effects_base: destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.416: 	spectrum.cpp:75	sie: spectrum destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.417: 	output_level.cpp:40	sie: output_level destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.420: 	stream_output_effects.cpp:134	destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.420: 	effects_base.cpp:152	effects_base: destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.421: 	spectrum.cpp:75	soe: spectrum destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	output_level.cpp:40	soe: output_level destroyed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:1661	Destroying PipeWire registry...
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:1664	Disconnecting PipeWire core...
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Sink 87 alsa_output.pci-0000_09_00.1.hdmi-stereo has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Sink 91 alsa_output.pci-0000_0b_00.3.iec958-stereo has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Source 92 alsa_input.pci-0000_0b_00.3.analog-stereo has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Sink 89 alsa_output.usb-0ccd_0077-00.analog-stereo has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Source 88 alsa_input.usb-0ccd_0077-00.mono-fallback has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Source 90 alsa_input.usb-046d_BRIO_4K_Stream_Edition_D4E1A92C-02.analog-stereo has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Stream/Output/Audio 225 Chromium has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Stream/Input/Audio 243 Chromium input has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Sink 121 easyeffects_sink has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:265	Audio/Source/Virtual 44 easyeffects_source has been removed
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:1669	Stopping PipeWire's loop...
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.422: 	pipe_manager.cpp:1672	Destroying PipeWire's context...
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.423: 	pipe_manager.cpp:1675	Destroying PipeWire's loop...
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.423: 	application.cpp:450	Shutting down...
(easyeffects:5241): easyeffects-DEBUG: 19:42:09.424: 	easyeffects.cpp:71	Exitting the main function with status: 0
```
</details>

### Additional Information

This seems to already have been noticed here https://github.com/wwmm/easyeffects/issues/1617, however, its not clear whether this is intended behavior (I at least couldn't find anything, and if intended a warning in the log would be nice).

This currently poses an issue when using home-on-tmpfs, where all settings are essentially discarded after a reboot.

**Comment by wwmm** [comment]
>Running easyeffects --gapplication-service -l my-preset doesn't load the preset my-preset

That is not how this is supposed to be used. After that you call `easyeffects -l my-preset`.

**Comment by Janrupf** [comment]
Is there a specific reason why this can't be supported? Would be nice to have it mentioned somewhere at least, I discovered this because the nix home-manager attempts to use it that way.

Alternatively, if it's just not implemented I could take a look into making it work if wanted.

**Comment by wwmm** [comment]
>Is there a specific reason why this can't be supported? Would be nice to have it mentioned somewhere at least, I discovered this because the nix home-manager attempts to use it that way.

I do not know why nix is trying to use it this way. The output of `easyeffects -h` does not suggests it would work. The reason why mixing things is not a good idea is that `--gapplication-service` is an option provided by gtk itself. It is not something handled by EasyEffects code.

Do you have any special reason to want to do this? IF you are always loading the same preset for the same device it is easier to set an autoloading profile.

**Comment by Janrupf** [comment]
> I do not know why nix is trying to use it this way.

I think they just saw that `--load-preset` exists and assumes it would work with `--gapplication-service`, much like I did.

> The output of easyeffects -h does not suggests it would work.

Running `easyeffects --help-all` kind of does, but that's more of a GTK issue than an easyeffects issue I suppose. I also checked the [GTK documentation](https://wiki.gnome.org/HowDoI/GtkApplication/CommandLine) about the service flag, and it somewhere states that it only enters service mode if `--gapplication-service` is the sole argument - looks like this is not true, it just ignores other arguments.

All in all, this seems to be more of a GTK issue than an easyeffects issue.

> Do you have any special reason to want to do this? IF you are always loading the same preset for the same device it is easier to set an autoloading profile.

Yeah, probably - from what I've seen in the code I could also manipulate the settings via `dconf` before starting easyeffects. The reason I want to do this is because effectively all settings are wiped each time I reboot (see home-on-tmpfs), as I want to configure them through nix.

I'm currently using a systemd service which just executes `easyeffects -l my-preset` in `ExecStartPost`, but I'm not sure if this is a race condition (as in, if there is a way this might execute earlier than `easyeffects --gapplication-service` in `ExecStart`). An autoload preset could also solve this issue.

I think it would be nice to actually have a way to load a preset when starting the application as a GApplication service, but the GTK documentation on passing commandline flags in this case is rather non-existent. This also rules out printing a warning, since I didn't see a simple way to hook into this mechanism at all.

So its probably more of a nice-to-have, but given the way GTK `--gapplication-service` this may not be easy.

**Comment by wwmm** [comment]
>The reason I want to do this is because effectively all settings are wiped each time I reboot (see home-on-tmpfs), as I want to configure them through nix.

Hum... That is definitely an unusual case. One of the reasons why I used gsettings is its ability to restore all the settings when we bind the widgets to the database properties. But with the database being gone on every reboot this makes the whole approach useless indeed...

>but I'm not sure if this is a race condition (as in, if there is a way this might execute earlier than easyeffects --gapplication-service in ExecStart). An autoload preset could also solve this issue.

If it were executing earlier the process would not enter service mode. So most likely it is being executed too soon. Before the code that has to process the preset loading is ready to do so. Maybe a delay of 3 or 5 seconds before launching the second command  will help.
